aa_stripe/signals.py

```diff
@@ -0,0 +1,10 @@
+def stripe_webhook_post_save(sender, instance, created, **kwargs):
```

Comment:
Still, I want parse function there to avoid future spaghetti code. Functions should do one thing and this will lead to mess in no time.